components/log-viewer-webui/server/src/main.ts

```diff
@@ -1,35 +1,71 @@
-import process from "node:process";
+// Reference: https://github.com/fastify/demo/blob/main/src/server.ts
 
-import app from "./app.js";
-import {
-    ENV_TO_LOGGER_MAP,
-    parseEnvVars,
-} from "./utils/env.js";
+import closeWithGrace from "close-with-grace";
+import fastify from "fastify";
+import fp from "fastify-plugin";
 
+import serviceApp from "./fastify-v2/app.js";
+
+
+const DEFAULT_FASTIFY_CLOSE_GRACE_DELAY = 500;
+const DEFAULT_PORT = 3000;
+
+/**
+ * Do not use NODE_ENV to determine what logger (or any env related feature) to use
+ *
+ * @return Logger options for Fastify.
+ * @see {@link https://www.youtube.com/watch?v=HMM7GJC5E2o}
+ */
+const getLoggerOptions = () => {
+    // Only if the program is running in an interactive terminal.
+    if (process.stdout.isTTY) {
+        return {
+            level: "info",
```

Comment:
Is this better? e.g., users wouldn't have to modify any VCS tracked file to change the log level
```suggestion
            level: process.env.LOG_LEVEL ?? "info",
```

Reply:
I also changed the default log level from silent to info for production. This is how we had it before.